beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactory.java

```diff
@@ -90,6 +90,8 @@ public Consumer<BeaconBlockBodyBuilder> createSelector(
     return bodyBuilder -> {
       final Eth1Data eth1Data = eth1DataCache.getEth1Vote(blockSlotState);
 
+      final UInt64 minimalViableEpoch = spec.computeMinimumViableEpoch(blockSlotState.getSlot());
```

Comment:
nit: Probably rename this to `previousForkEpochStart`? Just want to try and explain why it's the minimum viable.